nixos/modules/services/networking/mullvad-vpn.nix

```diff
@@ -14,6 +14,15 @@ with lib;
       '';
     };
 
+    enableExcludeWrapper = mkOption {
+      type = types.bool;
+      default = true;
```

Comment:
IMVHO this should not default to true.

Reply:
I kept this as `true` by default because it's needed for full functionality, and most users would expect it to just work after enabling the service. This default is the least surprising to the majority of users.

I do understand the other side though, requiring an explicit opt-in could be better for more security conscious users, but I don't feel strongly either way.

Reply:
Fair point.